scripts/tools/zap/run_zaptool.sh

```diff
@@ -39,6 +42,18 @@ function _print_usage() {
         echo " General Options:"
         echo ""
         echo "  -h, --help                  Display this help, then exit."
+        echo "  --zap ZAP                   Use the zap program ZAP as he zap executable to "
+        echo "                              run (default: $ZAP_CMD_DEFAULT)."
+        echo ""
+        echo " Some influential environment variables:"
```

Comment:
The key part is this: if either of these is used the `--zap` option will be ignored, right?  We should clearly document that interaction.

Reply:
Good feedback, thanks. Addressed and pushed.